src/store/dynamo.js

```diff
@@ -39,13 +39,28 @@ const getDumpId = ({ clientId }) => `${clientId}.gz`;
  *
  * @param {*} data
  */
-async function saveEntry({ ...data }) {
+async function saveEntry(data) {
     try {
-        const entry = Object.assign(data, {
-            dumpId: getDumpId(data),
-            conferenceId: data.conferenceId.toLowerCase(),
-            conferenceUrl: data.conferenceUrl?.toLowerCase()
-        });
+        const { conferenceId = 'undefined',
+            conferenceUrl = 'undefined',
+            userId = 'undefined',
+            app = 'undefined',
+            baseDumpId,
+            sessionId,
+            startDate,
+            endDate
+        } = data;
```

Comment:
There are cases where the identity entry doesn't go through (quick refreshes), and I think jvb connections also had some issues with providing this info all the time (not entirelty sure). But looking through the logs I saw some cases where only the userId was missing, I'm assuming this might be comming from clients with some custom configs like jibris, this could use some more investigation.